src/controllers/auth.js

```diff
@@ -1,11 +1,92 @@
+const jwt = require('jsonwebtoken');
+
+const bcrypt = require('bcrypt');
+
+const saltRounds = 10;
+
+const User = require('../models/user');
+
 module.exports = {
-  signInUser: (req, res) => {
-    res.send('You successfully signed in');
+  signInUser: async (req, res) => {
+    const { username, email, password, rememberMe } = req.body;
+    let cookieAge = 24 * 3600;
+    try {
+      const user = await User.findOne({
+        $or: [{ email: email }, { username: username }],
+      });
+      /* eslint object-shorthand: ["warn"] */
+      if (!user) throw new Error('Wrong username');
```

Comment:
Shouldn't this message be `Wrong username or email` since both can be used to login?